tests/integration_tests.rs

```diff
@@ -93,6 +93,144 @@ fn line_range_last_3() {
         .stdout("line 2\nline 3\nline 4\n");
 }
 
+#[test]
+fn tabs_passthrough_wrapped() {
+    bat()
+        .arg("tabs.txt")
+        .arg("--tabs=0")
+        .arg("--wrap=character")
```

Comment:
I think we can remove `--wrap`?

Reply:
ok! fixed in [f0f0672](https://github.com/sharkdp/bat/pull/356/commits/f0f0672ea922985d14e23e1bb94d892e6213107c)